nixos/default.nix

```diff
@@ -40,7 +40,7 @@ in
         after = [ "nix-daemon.socket" ];
 
         serviceConfig = {
-          User = username;
+          User = config.users.users.${username}.name;
```

Comment:
Might be slightly more idiomatic to use `usercfg.home.username` here.

Reply:
is that important to you? I like it this way a bit better, because this is how `home.username` is defined as well.